docs/components/court-cases-release-dates/latest-calculation-card.njk

```diff
@@ -43,7 +43,11 @@
             description: "Early Removal Scheme Eligibility Date",
             date: "2027-04-21"
         }
-    ]
+    ],
+    printNotificationSlip: {
+        href: '#',
+        dataQa: 'my-qa-hook'
```

Comment:
@yasinmustafa do you think we could do something a bit more meaningful than "my-qa-hook" "release-notification-hook"?